src/server/routes/rest.js

```diff
@@ -65,6 +66,39 @@ router.get('/gene-query', (req, res) => {
   });
 });
 
+//expose a rest endpoint for enrichment
+router.get('/enrichment', (req, res) => {
+  const genes = req.query.genes;
+  const tmpOptions = {};
+  tmpOptions.output = req.query.output;
```

Comment:
1. For each of these options is there some sort of default value?  What does gprofiler do if any of these are undefined?

2. It might also make sense for users to be able to POST with an options JSON body.  It might make more sense than having this as a GET request.

Reply:
1. Yes. It uses default value when the parameter is undefined. https://github.com/PathwayCommons/app-ui/pull/493/commits/f730ecf54a56c5f8ab08fb234cb76203a4c02ef1#diff-0fd263785891b1a6370048fe62b65e6bL92
https://github.com/wxli0/app-ui/blob/enrichment-endpoint/src/server/enrichment-map/enrichment/enrichment.js#L49